doc/en/components/calendar.md

```diff
@@ -0,0 +1,384 @@
+---
+title: $Platform$ Calendar | Calendar | Infragistics
+_description: Create an intuitive calendar for applications to display date information, and users to input date information with $ProductName$ Calendar component.

```

Comment:
The second part ("... and users to input ..." sounds strange.

Reply:
That's a copy of the Angular description. I guess that sometimes in order to have a better SEO, we sacrifice the human readability of the descriptions.